frb_codegen/src/config/raw_opts_parser.rs

```diff
@@ -1,13 +1,19 @@
 use crate::config::raw_opts::RawOpts;
-use anyhow::Context;
 use clap::Parser;
-use std::fs;
 
 impl RawOpts {
+    #[cfg(not(feature = "serde"))]
+    pub fn try_parse_args_or_yaml() -> anyhow::Result<Self> {
+        Self::try_parse().map_err(|err| err.exit())
+    }
+
     /// Parses options from arguments, or from a config file if no arguments are given.
     /// Terminates the program if argument validation fails.
     #[cfg(feature = "serde")]
     pub fn try_parse_args_or_yaml() -> anyhow::Result<Self> {
+        use anyhow::Context;
+        use std::fs;
+
```

Comment:
uber-nit: Not very sure, shall we do something like

```
pub fn try_parse_args_or_yaml() -> anyhow::Result<Self> {
  #[cfg(feature = "serde")]
  {
      ...
  }

  #[cfg(not(feature = "serde"))]
  {
      ...
  }
}

Reply:
Sure, that works too 👍